### PR TITLE
Update documentation and templates

### PR DESCRIPTION
**Pull request by dannycolin**
tl;dr of the changes:

* Move all development documentations in CONTRIBUTING.md
* Move maintainer documentations in /docs
* Add a Pull Request template
* Remove gdoc links in favor of SUMO documentation on MAC and containers

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Major change (fix or feature that would cause existing functionality to work differently than in the current version)
+
+Tag issues related to this pull request:
+
+*
+*
+*
```
Comment (maxxcrawford) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:24]: praise: Love this revised template. Clean and straightforward.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,35 +1,44 @@
 # Contributing
 
-Everyone is welcome to contribute to containers. Reach out to team members if you have questions:
+## Requirements
 
-- Matrix chat: [#containers:mozilla.org](https://matrix.to/#/#containers:mozilla.org)
-- Email: containers@mozilla.com
+* Firefox 91.1.0+
+* Git 2.13+
+* Node 7+
```
Comment (maxxcrawford) [inline CONTRIBUTING.md:7]: Chore: We should upgrade our node version at some point. We did this for recently for the [Relay add-on](https://github.com/mozilla/fx-private-relay-add-on/blob/main/package.json#L9) and also bumped up web-ext.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,35 +1,44 @@
...
+2. Fetch the locales:
 
-Check first if the bug might already exist: https://github.com/mozilla/multi-account-containers/issues
+    ```
+    cd multi-account-containers
+    git submobule update --init
+    ```
+3. Install the project dependencies
+    ```
+    npm install -legacy-peer-deps
```
Comment (maxxcrawford) [inline CONTRIBUTING.md:20]: Question: Does this need an extra `-`? 

```suggestion
    npm install --legacy-peer-deps
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,35 +1,44 @@
...
+    ```	
+3. Install [web-ext][web-ext] command-line tool:
 
-[Open an issue](https://github.com/mozilla/multi-account-containers/issues/new)
+    ```
+    npm install -g web-ext
```
Comment (maxxcrawford) [inline CONTRIBUTING.md:25]: Suggestion: Let's remove making install line. We can just have users install deps and run the [`npm run web-ext`](https://github.com/mozilla/multi-account-containers/blob/main/package.json#L39) command from the `package.json`

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,35 +1,44 @@
...
 
-1. Visit about:support
-2. Click "Copy raw data to clipboard" and paste into the bug. Alternatively copy the following sections into the issue:
-  - Application Basics
-  - Nightly Features (if you are in nightly)
-  - Extensions
-  - Experimental Features
-3. Include clear steps to reproduce the issue you have experienced.
-4. Include screenshots if possible.
+4. Run `web-ext run -s src/`.
```
Comment (maxxcrawford) [inline CONTRIBUTING.md:28]: See above comment

```suggestion
4. Run `npm run web-ext`.
```

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
WOW! This is a major improvement to getting started on the project. I have a few notes below. 😎

File: README.md [hunk 6]
```diff
@@ -2,110 +2,31 @@
...
-
-### Links
-
-Facebook & Twitter icons CC-Attrib https://fairheadcreative.com.
-
-- [License](./LICENSE.txt)
-- [Contributing](./CONTRIBUTING.md)
-- [Code Of Conduct](./CODE_OF_CONDUCT.md)
+<!-- Please keep the list in alphabetical order -->
+[contributing]: CONTRIBUTING.md
```
[the remark this answers is not in the document]
Comment (dannycolin) [reply README.md:28]: Yes. You can try it on https://github.com/dannycolin/multi-account-containers/tree/clean-documentation

File: CONTRIBUTING.md (hunk 5 above)
Comment (dannycolin) [reply CONTRIBUTING.md:28]: Actually, I was wondering if we shouldn't simply drop a `web-ext-config.js` file in the root folder so `web-ext` will work in both the root folder and `/src`.

File: CONTRIBUTING.md (hunk 4 above)
Comment (dannycolin) [reply CONTRIBUTING.md:25]: So you'd prefer the user to use the local `web-ext` bin than the global one right?

Comment (maxxcrawford) [reply CONTRIBUTING.md:25]: Agreed!

File: CONTRIBUTING.md (hunk 2 above)
Comment (dannycolin) [reply CONTRIBUTING.md:7]: Agreed. There's a few other deps that could be updated and/or I have no idea if we still need them. I can look at it in a follow-up bug ;)

**Review by maxxcrawford — APPROVED** [review]
Awesome work! Thanks!

**Pull request merged**